GEOSldas_App/sample_config_files/LADAS/exeinp.txt.Hy4dEnVar.central

```diff
@@ -7,16 +7,27 @@
 # (1) Create exeinp template using:
 #         ldas_setup sample --exeinp > MY_exeinp.txt
 #
-# (2) Use the resource parameter settings below when editing MY_exeinp.txt
+# (2) Use the resource parameter settings below when editing MY_exeinp.txt 
 #
+# (3) The following resource parameters are provided via fvsetup,
+#     which supersede the default in MY_exeinp.txt
+#      ADAS_EXPDIR
+#      ADAS_EXPID
+#      EXP_ID
+#      MET_TAG       
+#      BEG_DATE 
+#      GID 
 ##############################################################################
 
+# e.g., CENTRAL_AGCM_GRID = CF0360x6C
+EXP_DOMAIN:        [CENTRAL_AGCM_GRID]_GLOBAL 
+
 LADAS_COUPLING:    1
 
-ADAS_EXPDIR:       [full_path]/[ADAS_EXPDIR]
+ADAS_EXPDIR:       [full_path]/[ADAS_EXPID]
 
-MET_TAG:           [ADAS_EXPID]__Nx+- 
-MET_PATH:          ../../scratch  
+MET_TAG:           [ADAS_EXPID]__bkg 
```

Comment:
How do EXP_DOMAIN, ADAS_EXPDIR, and MET_TAG get assembled?  Is this done by fvsetup?  If so, we should add a comment that describes this. 
But, if we can implement the above suggestion (make ldas_setup use hardcoded rc variables for MET_HINTERP etc, do we even still need the present sample file?  Rather, it may make more sense to add a description of the process into a file called something like README_LADAS_setup

Reply:
ADAS_EXPDIR is a command line argument,  the others get assembled in ldas_setup. 
if we make a function in ldas_setup to handle all the default values and assembling, no need sample files.

Reply:
Ok, sounds good. But we still need to have a brief description of the handling of the various rc variables somewhere.